code/datums/supplypacks/security.dm

```diff
@@ -1,15 +1,6 @@
 /decl/hierarchy/supply_pack/security
 	name = "Security"
 
-/decl/hierarchy/supply_pack/security/specialops
```

Comment:
why remove smokebombs and incendiary stuff, are they broken?

smoke bombs could be pretty useful

Reply:
This crate just seemed off the cuff and nonsensical. It's also never ordered nor utilized. I've not particularly tested this crate or its contents out, but these are Bay legacy items, and I'm wanting to go on a limb here and say that we didn't alter them.

Reply:
eh fine, it'd be nice to see smokebombs readded somewhere in future, but not a huge issue